### PR TITLE
 Boot up prod env

### PR DESCRIPTION
**Pull request by ismail-shahzad**
This PR boots up a prod environment for WRI ODP

NOTE: Please Squash and Merge when merging

### CODE REVIEW

File: ckan-deployment/helm-templates/values.yaml.prod.template [hunk 1]
```diff
@@ -29,38 +26,34 @@ ckan:
...
+      CKAN__SITE_TITLE: WRI Open Data Platform
       CKAN__STORAGE_PATH: /tmp
       CKAN__VIEWS__DEFAULT_VIEWS: recline_view text_view image_view pdf_view geojson_view
-      CKAN__PLUGINS: image_view text_view webpage_view resource_proxy datatables_view datastore datapusher activity envvars
+      CKAN__PLUGINS: image_view text_view webpage_view resource_proxy datatables_view datastore datapusher activity s3filestore envvars
       CKAN__AUTH__CREATE_UNOWNED_DATASET: "True"
-      CKAN__AUTH__ALLOW_DATASET_COLLABORATORS: "True"
-      CKAN__AUTH__ALLOW_ADMIN_COLLABORATORS: "True"
-      CKAN__AUTH__ALLOW_COLLABORATORS_TO_CHANGE_OWNER_ORG: "True"
+      CKAN__AUTH__ALLOW_DATASET_COLLABORATORS: True
```
Comment (mpolidori) [inline ckan-deployment/helm-templates/values.yaml.prod.template]: I think these quotes need to be added back in (you fixed this previously).

**Review by mpolidori — CHANGES_REQUESTED** [review]

File: ckan-deployment/helm-templates/values.yaml.prod.template (hunk 1 above)
Comment (ismail-shahzad) [reply ckan-deployment/helm-templates/values.yaml.prod.template]: Good catch.
Ill add the quotes

Comment (ismail-shahzad) [reply ckan-deployment/helm-templates/values.yaml.prod.template]: Resolved

**Review by mpolidori — APPROVED** [review]
Looks good to me

**Review by dmannarino — APPROVED** [review]
Looks good!

**Pull request merged**